easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb

```
@@ -22,6 +22,7 @@ checksums = ['a8243923ee08c74cab609a4b086ade66c6156fc2b24450e2a500108dc644c867']
 builddependencies = [
     ('Autoconf', '2.69'),
     ('pkg-config', '0.29.2'),
+    ('Automake', '1.16.2'),
```

As we need both `Autoconf` and `Automake`, it's better to switch to `Autotools` (which contains both):
```suggestion
    ('Autotools', '20200321'),
    ('pkg-config', '0.29.2'),
```

---

Also, please make the same change to `PhyML-3.3.20190321-foss-2018b.eb`, to keep things consistent.

---

Done. I have done the replacement as suggested, which makes more sense, and I have updated the `PhyML-3.3.20190321-foss-2018b.eb` file as requested well.

---

Great, thanks @sassy-crick!